kythe/docs/schema/schema.txt

```diff
@@ -1201,6 +1201,33 @@ public class E {
 }
 --------------------------------------------------------------------------------
 
+[[refid]]
+ref/id
+~~~~~~~~~~~
+Brief description::
+  A *ref/name* B if A does not *ref* or *define* B but A's contents are
```

Comment:
`ref/id`?

Reply:
Whoops, this went through a couple of revisions.